### ISSUE TITLE
Add Google analytics to gitrank.io

### ISSUE THREAD

**Issue by callicles** [gitlinks/gitrank-web#95]
It would be nice to have some analytics on the plateform

**Comment by jacobtardieu** [comment]
Working on it.

**Comment by jacobtardieu** [comment]
Seems to work in prod! I'll see to put Content Square tag next week for more analytics (GA is blocked by ublock).
